.gitlab-ci.yml

```diff
@@ -254,6 +259,17 @@ dev-test-firefox:
     - .dev
     - .test-firefox
 
+dev-test-firefx-experiment:
```

Comment:
Ignore this part, it was for running the tests for this branch. I'll remove before we merge.